### DISCUSSION TITLE
Configuration task SMAC basline

### DISCUSSION THREAD

**Discussion by th-yoon** [ds4dm/ml4co-competition#57] [Q&A] [answered]
Hi, I have some questions about SMAC baseline on configuration task.
1.  In which criteria did you choose the target set of parameters?
2. For how long - # of evaluations and time limit - did you run the baseline code?
3. How many instances did you use to get the SMAC baseline result?

**Comment by antoniach** [comment, marked as answer]
Hi @th-yoon,

1. To build the set of parameters used for tuning in the baseline, we selected parameters that for us, seemed most crucial to control. This is by no means the perfect selection, but rather a starting point for building your own parameter set.

2. We ran SMAC for 10 iterations per instances and used a time limit of 300 seconds.

3. We used 10 instances for training.

The settings we used for training SMAC are the default settings of the baseline. A description can be found under Point 3 of `README.md` or in lines 97-124 of `run_training.py`.

I hope this answers all your questions.

Best,
Antonia

**Reply by th-yoon** [reply]
Much appreciated @antoniach !
